src/common/components/wrappers/Row.tsx

```diff
@@ -5,7 +5,7 @@ interface Props {
   marginBottom?: string;
 }
 
-export const ButtonRow = styled.div<Props>`
+export const Row = styled.div<Props>`
```

Comment:
Tror du må rebase, mener jeg brukte ButtonRow i min pr/commit.

Reply:
Endrer ButtonRow til Row, tenkte det var bedre med en generell rad-greie enn kun for knapper :)